run/idp-sql/e2e_test.py

```diff
@@ -18,6 +18,7 @@
 import json
 import os
 import subprocess
+from datetime import date
```

Comment:
Can we use `datetime.datetime` instead? https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#datetimedatetime-objects